src/main/java/com/beautydeals/demo/database/BasicSearching.java

```diff
@@ -0,0 +1,42 @@
+package com.beautydeals.demo.database;
+
+import java.sql.*;
+
+public class BasicSearching {
+    private static String tbl = "product_tbl";
+    private static String drv = "com.mysql.cj.jdbc.Driver";
+    private static String url = "jdbc:mysql://127.0.0.1:3306/BEAUTYDEALS";
+    private static String usr = "root";
```

Comment:
These configurations will change to the current database setting later:
- Database URL: 
`mysql://b6c467462f4954:38ab0380@us-cdbr-east-02.cleardb.com/heroku_7c76701c5a9944c?reconnect=true`
  - username: `b6c467462f4954`
  - password: `38ab0380`
  - host: `us-cdbr-east-02.cleardb.com`
  - database: `heroku_7c76701c5a9944c`

Reply:
Done!

Reply:
This database is just a temporary one for testing, maybe we can use a more general database for our application. We can discuss it later!